pysb/simulator/base.py

```diff
@@ -533,15 +559,35 @@ class SimulationResult(object):
     13.333333  0.000002    4.999927
     """
     def __init__(self, simulator, tout, trajectories=None,
-                 observables_and_expressions=None,
-                 squeeze=True,
-                 simulations_per_param_set=1):
-        simulator._logger.debug('SimulationResult constructor started')
+                 observables_and_expressions=None, squeeze=True,
+                 simulations_per_param_set=1,
+                 model=None, initials=None, param_values=None):
+        if simulator:
+            simulator._logger.debug('SimulationResult constructor started')
+            self._param_values = simulator.param_values.copy()
+            try:
+                self._initials = simulator.initials.copy()
+            except SimulatorException:
+                # Network free simulations don't have initials list, only dict
+                self._initials = simulator.initials_dict.copy()
+            self._model = copy.deepcopy(simulator._model)
+            self.simulator_class = simulator.__class__
+            self.init_kwargs = copy.deepcopy(simulator._init_kwargs)
+            self.run_kwargs = copy.deepcopy(simulator._run_kwargs)
+        else:
+            self._param_values = param_values
+            self._initials = initials
+            self._model = model
+            self.simulator_class = None
+            self.init_kwargs = {}
+            self.run_kwargs = {}
+
         self.squeeze = squeeze
         self.tout = np.asarray(tout)
         self._yfull = None
-        self._model = simulator._model
         self.n_sims_per_parameter_set = simulations_per_param_set
+        self.pysb_version = PYSB_VERSION
+        self.creation_date = datetime.now()
```

Comment:
Minor nitpick, but this isn't just a date. Would `timestamp` be an acceptable name?